app/models.py

```diff
@@ -995,7 +1000,7 @@ def serialize(self):
             "created_by": self.created_by.email_address,
             "version": self.version,
             "body": self.content,
-            "subject": self.subject if self.template_type != SMS_TYPE else None,
+            "subject": self.subject if self.template_type in {EMAIL_TYPE, LETTER_TYPE} else None,
```

Comment:
why is it curly braces here, not the square braces?

Reply:
curly is a `set` rather than a `list`. technically it's marginally quicker to lookup an item in a set rather than in a list (constant time, rather than dependent on the length of the list). In practice in this example it'll make little to no difference

Reply:
Ahh, ok, thanks! :)